### ISSUE TITLE
[Feature Request] Demo Recording  !

### ISSUE THREAD

**Issue by TP555** [luciusDXL/TheForceEngine#239]
Hi

Can you add Recording for Time/Demos like in Gzdoom or Qauke Series Games  ?

**Comment by luciusDXL** [comment]
I have plans for something like this eventually, but it is low priority for now.

**Comment by mlauss2** [comment]
This could also be useful for PGO builds.  TFE is already fast enough on contemporary hardware, but I've seen PGO speed up certain programs by almost a magnitude, which could also help slower systems (I'm thinking here about my software-renddering-only 400-MHz MIPS32 systems).

**Comment by TP555** [comment]
@mlauss2

Yeah good point , but for Retro Hardware (32Bit CPU's) , we need first a 32Bit TFE , at this momment it is only for 64Bit  !

**Comment by mlauss2** [comment]
32bit works, as I wrote, I run it on MIPS32 FPU-less hardware.

**Comment by TP555** [comment]
@mlauss2

ok , i saw not now in the Changelogs , that TFE now runs with 32Bit OS/CPU's !
But good to know  !

**Comment by luciusDXL** [comment]
This has been implemented in version 1.22 - called Replays.
